test/frontend/components/coded_set_table_form_test.js

```diff
@@ -19,7 +19,7 @@ describe('CodedSetTableForm', () => {
   });
 
   it('should create the table', () => {
-    expect(component.find("tr").length).to.equal(5);
+    expect(component.find("tr").length).to.equal(4);
```

Comment:
I'm not about to reject this PR, but I do think we should move away from these hard coded values. When I changed the fixtures myself I had to chase down errors in other people's tests as a result.
Just something to start doing going forward in future sprints maybe?